Source/WebCore/platform/mediarecorder/cocoa/AudioSampleBufferCompressor.mm

```diff
@@ -307,54 +318,39 @@
     return retainPtr([NSNumber numberWithInt:(primeInfo.leadingFrames / m_destinationFormat.mFramesPerPacket)]);
 }
 
-RetainPtr<CMSampleBufferRef> AudioSampleBufferCompressor::sampleBuffer(AudioBufferList fillBufferList)
+Expected<RetainPtr<CMSampleBufferRef>, OSStatus> AudioSampleBufferCompressor::sampleBuffer(AudioBufferList* fillBufferList, uint32_t numSamples)
```

Comment:
Why do we have to pass fillBufferList as an in/out parameter?
Can we simplify and just pass a const WebAudioBufferList&?

Reply:
We don't, but this is your code so I left that part as is :)